stable/prometheus-cloudwatch-exporter/templates/configmap.yaml

```diff
@@ -8,5 +8,5 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 data:
-  config.yaml: |
-{{ printf .Values.config | indent 4 }}
\ No newline at end of file
+  config.yml: |
```

Comment:
The entrypoint in the Docker image expects `config.yml` instead of `config.yaml` as per https://github.com/prometheus/cloudwatch_exporter/blob/master/Dockerfile#L18